privilege/privileges/privileges.go

```diff
@@ -184,10 +186,12 @@ func (p *UserPrivileges) ConnectionVerification(user, host string, authenticatio
 	}
 
 	pwd := record.AuthenticationString
-	//if len(pwd) != 0 && len(pwd) != mysql.PWDHashLen+1 {
-	//	logutil.BgLogger().Error("user password from system DB not like sha1sum", zap.String("user", user))
-	//	return
-	//}
+
+	// The length of the password encrypted by MD5 is 35
+	if len(pwd) != 0 && len(pwd) != 35 {
```

Comment:
don't hardcode the magic number, please use a static variable to keep it instead.